css/properties/mask-image.json

```diff
@@ -46,7 +46,9 @@
               "notes": "Initially, Safari supported only <code>-webkit-</code> prefixed values for gradients (such as <code>-webkit-linear-gradient()</code>). Later, support for unprefixed values was added."
             },
             "samsunginternet_android": {
-              "version_added": null
+              "version_added": "1.0",
+              "prefix": "-webkit-",
+              "notes": "From version 8, Chrome added support for gradient values. Initially, Chrome supported only <code>-webkit-</code> prefixed values for gradients (such as <code>-webkit-linear-gradient()</code>). Later, support for unprefixed values was added."
```

Comment:
Another note we can nuke, as it talks about support from before Samsung Internet 1.0 (and even Chrome Android's first release) were around.

```suggestion
```